analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RoslynSymbolicExecution.cs

```diff
@@ -55,7 +55,7 @@ public RoslynSymbolicExecution(ControlFlowGraph cfg, SymbolicCheck[] checks, Can
             {
                 throw new ArgumentException("At least one check is expected", nameof(checks));
             }
-            this.checks = new(new[] { new ConstantCheck() }.Concat(checks).ToArray());
+            this.checks = new(new SymbolicCheck[] { new ConstantCheck(), new InvocationCheck() }.Concat(checks).ToArray());
```

Comment:
This would allows to return a single state from
```cs
public override ProgramState[] PreProcess(SymbolicContext context) =>
    context.SetOperationConstraint(BoolConstraint.True);
```